modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc

```diff
@@ -13,21 +13,21 @@ Before using the {product-title} (ROSA) CLI, `rosa`, to create {hcp-title-first}
 * You have completed the AWS prerequisites for {hcp-title}.
 * You have available AWS service quotas.
 * You have enabled the ROSA service in the AWS Console.
-* You have installed and configured the latest ROSA CLI (`rosa`) on your installation host.
-+
-[NOTE]
-====
-To successfully install {hcp-title} clusters, use the latest version of the ROSA CLI (`rosa`).
-====
+* You have installed and configured ROSA CLI (`rosa`) version 1.2.23 or later on your installation host.
 * You have logged in to your Red Hat account by using the ROSA CLI.
 
 .Procedure
 
-. If they do not exist in your AWS account, create the required account-wide STS roles and policies by running the following command: 
+. If they do not exist in your AWS account, create the required account-wide STS roles and policies by running the following command:
```

Comment:
NP
Since the policies already exist in each AWS account and are managed by AWS, we are just attaching them to the roles, but not creating the policies.
```suggestion
. If they do not exist in your AWS account, create the required account-wide STS roles and attach the policies by running the following command:
```